CMakeLists.txt

```diff
@@ -186,7 +186,7 @@ int main() { }"
 HAVE_ATTRIBUTE_NOINLINE )
 
 IF ( HAVE_DECLSPEC_NOINLINE )
-  ADD_DEFINITIONS ( -D "PBRT_NOINLINE=__declspec(noinline)" )
+  ADD_DEFINITIONS ( -D "PBRT_NOINLINE=__declspec\\(noinline\\)" )
```

Comment:
Unfortunately this breaks the MSVC build. :-(. I'm curious why it matters for Cygwin. Would it work to check HAVE_ATTRIBUTE_NOINLINE first, under the theory that should also be true for cygwin?

Reply:
The problem is with `sh` which treats parentheses as piece of it's script. Checking if reorder helps.